blog/content/post/2022-09-21-qubes-os-summit-2022.md

```diff
@@ -0,0 +1,272 @@
+Qubes OS.
+
+In this presentation, a demo of Dasharo[1] distribution compatible with Alder
+Lake-S desktop MSI PRO Z690-A WIFI DDR4 running Qubes OS will be shown. The
+presenter will also describe new updates to Dasharo firmware and challenges
+awaiting in future development. Also it will be discussed how Dasharo plans to
+meet the future Qubes certification requirements[2] and approaches the openness
+of the firmware based on Dasharo Openness Score of various supported platforms.
+
+[Qubes OS: Towards Being a Reasonably Learnable System](https://youtu.be/A9GrlQsQc7Q?t=14609)
+held by Deeplow
+
+Following up on last year's presentation, deeplow presents the final work for
+his contribution proposal for an integrated onboarding tutorial for Qubes OS.
+
+[TrenchBoot - the only AEM-way to boot Qubes OS](https://youtu.be/A9GrlQsQc7Q?t=17441)
+held by Michał Żygowski
+
+The presentation will describe the project plan of improving and extending the
+Qubes OS AEM with TrenchBoot[5] covering both Intel and AMD hardware, TPM 1.2
+and 2.0. The goal is to unify the D-RTM early launch and Anti Evil Maid
+software to secure the Qubes OS boot process for basically any hardware device
+(as long as it supports the required technologies). The presenter will give
+detailed overview of project phases and tasks to be fulfilled as well as the
+cost outline. At the end a short demo of Qubes OS AEM with TrenchBoot on Dell
+OptiPlex 7010/9010 with Intel TXT and TPM1.2 will be shown.
+
+[Secure hardware for a secure operating system](https://youtu.be/A9GrlQsQc7Q?t=19620)
+held by Brent Cowing
+
+In this session we are going to discuss why you should care about open source
+firmware and the ways we can reduce the risk of firmware persistent malware.
+As the world is evermore consumed by detecting and preventing ransomware and
+other financially damaging attacks on systems and organizations, far too little
+attention has been paid to an attack surface common to every single
+vulnerability - firmware. It’s time for firmware to be open source and secure.
+
+[Design session: hardware requirements, firmware security](https://youtu.be/A9GrlQsQc7Q?t=21769)
+held by Marek Marczykowski-Górecki.
+
+
+[Dasharo vs vendor firmware performance on QubesOS - a comparison](https://youtu.be/A9GrlQsQc7Q?t=25694)
+held by Michał Kopeć.
+
+The presentation will compare performance between Dasharo and Vendor BIOSes
+on Dasharo-supported platforms, in the context of QubesOS usage. I will attempt
+to present the most significant differences that an end user user will see when
+installing Dasharo over the original proprietary firmware.
+
+In this presentation I will present actual difference in performance as measured
+by a set of benchmarks, talk about what causes them and what challenges there
+are to bring performance to a state that is competitive with the vendor BIOS.
+We'll also discuss how these differences may impact QubesOS users.
+
+
+## Hackaton
```

Comment:
```suggestion
## Hackathon
```
?

Ideally would be if some picture would be added.Of course respecting privacy.

Reply:
Fixed here and in a few more places, I've added link to the yt highlights from the hackathon.